src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -56,6 +55,8 @@ def find_or_create_branched_package
         return target_package if updated_pull_request? && target_package.present?
 
         branch
+      rescue StandardError => e
```

Comment:
This can be anything, not just failures in branching.

Reply:
That sounds to me more as a reason in favor of doing this change.

Reply:
It's like killing a fly with a sword. Sure, you could do it, but It's overkill. A lot of errors inherit from `StandardError`, so as an example of errors which I'm sure we don't want to rescue from in this `find_or_create_branched_package` method: [ArgumentError](https://ruby-doc.org/core-2.5.0/ArgumentError.html), [EncodingError](https://ruby-doc.org/core-2.5.0/EncodingError.html) and [ZeroDivisionError](https://ruby-doc.org/core-2.5.0/ZeroDivisionError.html) (there are much more...)

Instead, we should find out which errors might happen and rescue from those specific errors. If needed, it depends on the errors, it might be needed to provide a custom error message to make sure users understand why something went wrong.

Reply:
> Instead, we should find out which errors might happen and rescue from those specific errors.

What about "the other errors"? Shouldn't we report them too?

Reply:
Users won't be able to act on them, so it's not going to help them. It will also prevent errors from being sent to Errbit since they are caught, so we won't know about them. There could be bugs and we wouldn't know about them unless users go out of their way and warn us about them.

Reply:
So what do you propose?

Reply:
As I wrote before, find out which errors might happen and rescue from those specific errors.

Reply:
I've gone through the code, searching for the exceptions, and added a couple of `rescue` clauses. Please take a look again.